pkg/common/cns-lib/volume/manager.go

```diff
@@ -763,3 +765,55 @@ func (m *defaultManager) RelocateVolume(ctx context.Context, relocateSpecList ..
 	}
 	return res, err
 }
+
+// ConfigureVolumeACLs configures net permissions for a given CnsVolumeACLConfigureSpec
+func (m *defaultManager) ConfigureVolumeACLs(ctx context.Context, spec cnstypes.CnsVolumeACLConfigureSpec) error {
+	log := logger.GetLogger(ctx)
+	err := validateManager(ctx, m)
+	if err != nil {
+		return err
+	}
+	// Set up the VC connection
+	err = m.virtualCenter.ConnectCns(ctx)
+	if err != nil {
+		log.Errorf("ConnectCns failed with err: %+v", err)
+		return err
+	}
+
+	var task *object.Task
+	var taskInfo *vim25types.TaskInfo
+
+	task, err = m.virtualCenter.CnsClient.ConfigureVolumeACLs(ctx, spec)
```

Comment:
Does CNS handle idempotency for `ConfigureVolumeACLs` API? If not, we need to add it right?

Reply:
We don't need to handle idempotency for `ConfigureVolumeACLs`. Multiple requests with same spec result in same state for this API